root/server.js

```diff
@@ -18,24 +18,22 @@ var jwt             = require('jwt-simple');
 var app_middleware  = require('./app/middleware')
 var require_login   = app_middleware.require_login
 
-
 app.set('port', process.env.PORT || 3000);
 app.set('views', __dirname + '/views');
 app.set('view engine', 'jade');
 
+app.use(passport.initialize());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use(morgan('dev'));
 app.use(express.favicon());
-app.use(express.logger('dev'));
```

Comment:
These already initialised in app.use(morgan('dev')); (mogran is the new logger (name has changed))